types/slate/index.d.ts

```diff
@@ -1131,9 +1131,12 @@ export interface Plugin {
     schema?: SchemaProperties;
 }
 
+export type Plugins = Plugin | Plugin[] | PluginStack;
```

Comment:
I don't think Plugin[] is needed here because PluginStack covers it.
Is there any reason giving it a name 'stack'? How about CompositePlugin? Or, from the documentation, FeaturePlugin or FrameworkPlugin?

Reply:
> I don't think Plugin[] is needed here because PluginStack covers it.
Good catch. Thanks!

> Is there any reason giving it the name 'stack'?
Maybe there's a better name than `PluginStack`, but I surely wouldn't use any of the names you suggested as they all imply some kind of behavior, which at this point can be misleading bit of info. It's a stack as opposed to the readonly property on the controller, which is always a flat list, so more about the possible data structure than its content. Maybe `NestedPlugins`?